### ISSUE TITLE
Define the target users

### ISSUE THREAD

**Issue by dhprice** [robot-army/hotstick#19]
One way to categorise and qualify requirements is by target user/usage;

Hobbyist - usability and simplicity
Hackspace/Makerspace - ruggedness and safety
Professional - ?

**Comment by robot-army** [comment]
Yes, this, target users will define what standards we need to comply with as well. 

To narrow it down, I doubt my workplace would buy an open-source iron, so let's aim high and go for the hackspace user. I think the durability required by the hackspace would make it an attractive option for a home user/hobbyist as well.

**Comment by dhprice** [comment]
At this stage I would look to tag requirement and design choices with the usage class so the choice can be left to the builder. There may be requirements that would suit a hackspace but not be cost justifiable for a hobbyist, for example. If the first step in the analysis of requirements is to categorise them then the design rationale becomes easier.
